### PR TITLE
Take 4, syndie items  changes

### PR DESCRIPTION
**Pull request by enthethan**
## About The Pull Request
Im so tired.

## Why It's Good For The Game
just let it stop...

## Changelog
:cl:
tweak: X4 can now be planted on people
add: Adds rad bow, the ebows radiation inducing cousin for 8 Tc,it  irradiates and induces debilitating effects like brain damage and toxin damage. Unlike the ebow it doesn't have notable disabling power. Has a recharge rate of 25 seconds.
balance: Syndie cards now do 2 extra damage and can embed
balance: polonium now irradiates 2.5x faster than before but also metabolizes 2x faster
/:cl:

### CODE REVIEW

**Comment by qwertyquerty** [comment]
vote is very controversial

**Comment by enthethan** [comment]
the ebow change was controversial,i readded the old ebow while keeping the new one as radbow,will need a new sprite for the radbow as its using ebows sprite currently its not critical though and can be added in a later PR.

File: code/modules/projectiles/guns/energy/special.dm [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/obj/item/gun/energy/ionrifle
+ /obj/item/gun/energy/ionrifle
```
Comment (qwertyquerty) [inline code/modules/projectiles/guns/energy/special.dm]: remove space here

Comment (enthethan) [reply code/modules/projectiles/guns/energy/special.dm]: oops

**Pull request merged**